easy-rules-core/src/test/java/org/jeasy/rules/core/DefaultRulesEngineTest.java

```diff
@@ -56,7 +58,6 @@
     public void setup() throws Exception {
         super.setup();
         when(rule1.getName()).thenReturn("r");
-        when(rule1.getDescription()).thenReturn("d");
```

Comment:
I don't understand the removal of this. Is it related to the purpose of this PR?

Reply:
It was an IntelliJ warning. This mock isn't needed